src/officehours_ui/context_processors.py

```diff
@@ -22,21 +25,20 @@ def spa_globals(request):
         'first_name': request.user.first_name,
         'last_name': request.user.last_name,
     } if request.user.is_authenticated else None
-    backend_classes = {
-        backend_name: getattr(getattr(backends, backend_name), 'Backend')
+
+    backend_dicts: List[BackendDict] = [
+        getattr(getattr(backends, backend_name), 'Backend').get_public_data()
         for backend_name in settings.ENABLED_BACKENDS
-    }
+    ]
+
     return {
         'spa_globals': {
             'user': user_data,
             'feedback_email': getattr(settings, 'FEEDBACK_EMAIL', None),
             'debug': settings.DEBUG,
             'ga_tracking_id': settings.GA_TRACKING_ID,
             'login_url': settings.LOGIN_URL,
-            'backends': {
-                k: v.friendly_name
-                for k, v in backend_classes.items()
-            },
+            'backends': backend_dicts,
```

Comment:
Dict of dicts would be fine, but the serializers use arrays of objects in most places, so I guess I figured I'd go with this approach? In dict of dicts situations, I tend to wonder what the key should be, an unique `int` or `str`? It's convenient for indexing, but it always feels weird to repeat that data within the object and also feels wrong to leave it out. So that usually leaves me returning to array-of-dicts solutions.